src/fragments/lib/graphqlapi/js/setup/connect.mdx

```diff
@@ -285,25 +277,20 @@ export class AppModule { }
 </Block>
 
 <Block name="NextJS">
-  
-#### Install Amplify Libraries for NextJS
 
-The first step to using Amplify in the client is to install the necessary dependencies:
+The first step to using Amplify in the client for NextJS is to install the necessary dependencies:
 
 ```bash
 npm install aws-amplify @aws-amplify/ui-react
 ```
 
 The `aws-amplify` package is the main library for working with Amplify in your apps. The `@aws-amplify/ui-react` package includes React-specific UI components that you can use to build your app UI. To learn more about Amplify UI, visit our [documentation](https://ui.docs.amplify.aws/).
-[question]: #(RENE, Is there a more specific link to the UI docs that would be helpful to the user here? Is there a specific part of the docs they should be looking at for more info?)
 
```

Comment:
Q on the UI link here - is there a better link we can give them - is there a specific part of the docs they should be looking at?